Tests/NIOTests/NonBlockingFileIOTest.swift

```diff
@@ -915,4 +915,22 @@ class NonBlockingFileIOTest: XCTestCase {
             XCTAssertEqual(numberOfChunks, numberOfCalls.load())
         })
     }
+    
+    func testThrowsErrorOnUnstartedPool() {
+        
+        let fileURL = URL(fileURLWithPath: "/tmp/nonblockingfileiotest.txt")
```

Comment:
Can we use the `withTemporaryFile` helper here? It avoids test runs colliding with each other.